### ISSUE TITLE
No CORS when using iframe with youtubelinks

### ISSUE THREAD

**Issue by proProbe** [eisbehr-/jquery.lazy#57]
Trying to use the iframe plugin and have a embed youtubelink gives a CORS-error

```
No 'Access-Control-Allow-Origin' header is present on the requested resource
```

**Comment by eisbehr-** [comment]
Hello @proProbe and thanks for your feedback!

I assume you're using the `data-error-detect`, correct? If so, yes, there is bug in the actual version. I've missed to add the headers on the ajax request. I will fix this within the next version. You can disable the option, or even better, use the youtube plugin instead of the iframe plugin.

**Comment by proProbe** [comment]
Yes exactly, Ive been using the data-error-detect.
Im not sure I can actually use the youtube plugin as the iframe src itself could be other srcs than youtube. One example is a vimeo link

**Comment by eisbehr-** [comment]
This problem should be fixed as of version 1.7.2. There is even a Vimeo plugin now.
